draft-ietf-quic-ack-frequency.md

```diff
@@ -494,9 +494,15 @@ estimate of the typical RTT, but not the minimum RTT (min_rtt)
 unnecessarily high number of acknowledgments when min_rtt is much smaller than
 smoothed_rtt.
 
-Note that the congestion window and the RTT change over the lifetime of a
+Note that the congestion window and the RTT estimate change over the lifetime of a
 connection and therefore might require sending frequent ACK_FREQUENCY frames to
-ensure optimal performance.
+ensure optimal performance, though not every change should trigger an update.
+Usually, it is not necessary to send an ACK_FREQUENCY frame more than once per
+RTT and likely it needs to be sent even less frequently.
+Ideally, an ACK_FREQUENCY frame is sent only if a relevant change in the
+average congestion window or RTT estimates is detected that impacts the local
```

Comment:
```suggestion
Ideally, an ACK_FREQUENCY frame is sent only if a relevant change in the
longer-term development of the congestion window or RTT estimates is detected that impacts the local
```

Reply:
I'd prefer something like: "Ideally, an ACK_FREQUENCY frame ought to be sent after a change in the
the window or the smoothed RTT that impacts the local
setting of the reordering threshold or locally-selected calculation of the
Ack-Eliciting Threshold or Requested Max Ack Delay."

Reply:
```suggestion
Ideally, an ACK_FREQUENCY frame is sent only if a relevant change in the congestion window or RTT estimates is detected that impacts the local
```